src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.html

```diff
@@ -0,0 +1,7 @@
+<!--
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+-->
+<ul #mdtoolbar class="markdown-toolbar"></ul>
```

Comment:
nit: But for a super small one like this I prefer to just inline the definition in the ts file. Don't have to bother doing that here but for future reference.